src/index.js

```diff
@@ -67,8 +67,10 @@ const VLazyImageComponent = {
   mounted() {
     if ("IntersectionObserver" in window) {
       this.observer = new IntersectionObserver(entries => {
-        const image = entries[0];
-        if (image.isIntersecting) {
+        // Use `intersectionRatio` because of Edge 15's
+        // lack of support for `isIntersecting`.
+        // See: https://github.com/w3c/IntersectionObserver/issues/211
+        if (entries.some(e => e.isIntersecting || e.intersectionRatio > 0)) {
```

Comment:
Sometimes number of entries in `IntersectionObserver` callback is not 1 but 2. Both entries relate to the same element, but one have `isIntersected=false` and the other one `isIntersected=true`. This causes first intersection to be not captured.
I debugged the code, and `observe()` function is called only once, so I don't know how it's possible, that callback has 2 entries.
This I could easily reproduce in newest chrome.

Also, reading through code of `vue-lazy-hydrate` I found this line:
https://github.com/maoberlehner/vue-lazy-hydration/blob/master/src/utils.js#L13
So I added the same fix with `intersectionRatio`.